.github/CODEOWNERS

```diff
@@ -503,8 +503,8 @@
 /core/templates/services/app.service*.ts @srijanreddy98
 /core/templates/services/contextual/document-attribute-customization.service*.ts @srijanreddy98
 /core/templates/services/contextual/meta-tag-customization.service*.ts @srijanreddy98
-/core/templates/services/interaction-rules-registry.service*.ts @DubeySandeep
-/core/templates/services/interaction-specs.service*.ts @DubeySandeep
+/core/templates/services/interaction-rules-registry.service*.ts @oppia/lace-backend-reviewers
+/core/templates/services/interaction-specs.service*.ts @oppia/lace-backend-reviewers
```

Comment:
Should these be `@oppia/lace-frontend-reviewers` instead?

Reply:
fixed!